pkg/trace/stats/concentrator.go

```diff
@@ -48,6 +48,9 @@ type Concentrator struct {
 	agentVersion           string
 	peerSvcAggregation     bool // flag to enable peer.service aggregation
 	computeStatsBySpanKind bool // flag to enable computation of stats through checking the span.kind field
+	conf                   *config.AgentConfig
```

Comment:
We probably only need the subset of configuration that's relevant to the concentrator.